perun/cli_groups/import_cli.py

```diff
@@ -31,18 +37,18 @@
     help="Specifies the head minor version, for which the profiles will be imported.",
 )
 @click.option(
-    "--exitcode",
-    "-e",
+    "--stats-info",
+    "-t",
     nargs=1,
-    required=False,
-    default="?",
-    help=("Exit code of the command."),
+    default=None,
+    metavar="<stat1-description,...>",
+    help="Describes the stats associated with the imported profiles. Please see the import "
```

Comment:
Shouldn't this be file? If not, change to `[STR]` and write an example, how, the stats should look.